README.md

```diff
@@ -434,6 +435,13 @@ Before contributing be sure to consult the
 - [Stack Exchange]
 - [Reddit]
 
+# Spacemacs Everywhere
+
+Once you've learned Spacemacs key bindings, you can use them also in other IDEs/tools thanks to the following projects:
```

Comment:
- Add a "the" between `learned` and `Spacemacs`.
Remove "also".
Add a "," (comma) after `IDEs/tools`.
With those changes the sentence reads as:
>Once you've learned the Spacemacs key bindings, you can use them in other IDEs/tools, thanks to the following projects:

- Sort the links alphabetically.
- And sdwolfz suggested adding a link to: https://github.com/SpaceVim/SpaceVim